lib/services/saml.go

```diff
@@ -103,6 +103,10 @@ func ValidateSAMLConnector(sc types.SAMLConnector, rg RoleGetter) error {
 	if rg != nil {
 		for _, mapping := range sc.GetAttributesToRoles() {
 			for _, role := range mapping.Roles {
+				if strings.ContainsRune(role, '$') {
+					// Role is a template so we cannot check for existence of that literal name.
```

Comment:
Do you think a log (at debug level) would be helpful here?

Reply:
I think it might just produce more log spam than utility. Every time a SAML connector is validated (which happens a bit more often than necessary), if that connector uses a template name for a role, we'll log. Not sure that is going to help much with anything, although happy to be convinced otherwise.